include/EASTL/sort.h

```diff
@@ -707,6 +754,34 @@ namespace eastl
 	}
 
 
+	template <typename RandomAccessIterator, typename T, typename Compare>
+	inline RandomAccessIterator get_partition(RandomAccessIterator first, RandomAccessIterator last, const T&& pivotValue, Compare compare, typename eastl::enable_if<eastl::is_move_constructible<T>::value>::type* = 0 )
```

Comment:
Unfortunately, Github review tools don't make horizontal scrolling easy.  Please try to stay within a 120 column width.  I'd recommend using clang-format with our provided config file.